app/pods/components/web-components/au-abbreviated-text/template.hbs

```diff
@@ -1,20 +1,24 @@
 {{#if this.textTooLong}}
   <div class="auk-u-flex">
-    {{this.textAbbreviated}}
+    {{#if this.isCustom}}
+      <div class="{{@customTextClass}}">{{this.textAbbreviated}}</div>
+    {{else}}
+      {{this.textAbbreviated}}
+    {{/if}}
     <WebComponents::AuButtonLink
             @icon="circle-more"
             @skin="mute"
-            @layout="icon-only">
+            @layout="icon-only"
+            class="auk-u-flex--vertical-center"
```

Comment:
Nu flexed de knop veel beter in een hoge div, ipv bovenaan staat die nu mooi in het midden